e2e/tests/connect-and-status.spec.ts

```diff
@@ -7,12 +7,12 @@ test('check resource names and multiple statuses', async ({ page }) => {
   const resourceNames = page
     .getByTestId('resource-names')
     .getByTestId('resource-name');
-  await expect(resourceNames).toHaveCount(4);
+  await expect(resourceNames).toHaveCount(3);
```

Comment:
I'm not sure why the number of builtin resources is only 1 now, maybe something changed in the server implementation. It appears to be unrelated to my changes.